contrib/mbedtls/premake5.lua

```diff
@@ -16,4 +16,5 @@ project "mbedtls-lib"
 		"library/*.c"
 	}
 
-
+	filter { "system:windows or macosx" }
```

Comment:
Let's revert this file, since it would still create a .lib file that is empty...

I think just not linking against it is enough, since the dependency tree will not include it (at least in visual studio that will work)... Thank you for those modifications...